dask/dataframe/utils.py

```diff
@@ -504,9 +504,6 @@ def _check_dask(dsk, check_names=True, check_dtypes=True, result=None, scheduler
             )
             if check_dtypes:
                 assert_dask_dtypes(dsk, result)
-        else:
-            msg = f"Unsupported dask instance {type(dsk)} found"
-            raise AssertionError(msg)
```

Comment:
Is it necessary to remove this else block?

Reply:
Valid point.  I've added it back.